examples/app_commands/slash_options.py

```diff
@@ -15,3 +17,11 @@ async def hello(
     age: Option(int, "Enter your age", required=False, default=18),
 ):
     await ctx.send(f"Hello {name}")
+    
+@bot.slash_command(name="wish", description="Wish another member") # This will become a global slash command. However, global slash commands can take up to 1 hour to register.

```

Comment:
this line is violating some common python styling conventions: 2 spaces before a comment, max line width = 79. I think it is better to use pre-commit and enable linters and auto-styling tools like black so that you catch and fix these errors before merging the code in the master branch.